.travis.yml

```diff
@@ -10,6 +10,8 @@ before_install:
 
 install:
   - 'npm install'
+  - '(cd fluent; npm install)'
```

Comment:
If the aim is to push them as a single npm package I don't think these works does it? Surely we need to include the packages/dependencies into the main package.

Or is the plan to still release them as 2 separate npm packages going forward.

Reply:
We briefly discussed two npm packages.  "Option 3": https://github.com/kubernetes-client/javascript/issues/234.  Happy to re-discuss and change course now or later.

Now that I have experience across both code bases, I think we might eventually end up with a version of "Option 2".  One reason not to do Option 2 now is that it's another change for `godaddy/kubernetes-client` users.  I think starting with Option 3 and switching to Option 2 later will be relatively easy.  I don't feel strongly, however.